lib/aws/rails/middleware/elastic_beanstalk_sqsd.rb

```diff
@@ -8,6 +8,8 @@ class ElasticBeanstalkSQSD
         def initialize(app)
           @app = app
           @logger = ::Rails.logger
+
+          init_executor if ENV['AWS_PROCESS_BEANSTALK_WORKER_JOBS_ASYNC']
```

Comment:
minor nit (up to you) but you could do something like:
```
return if ENV[...]
@executor = executor

def executor
  Concurrent::...
end
```
so that it's clear in initialization what instance variables may exist.

Reply:
yea, good idea.  Changed.